azure-app-service/massdriver.yaml

```
@@ -0,0 +1,181 @@
+schema: draft-07
+name: <md .Name md>
+description: <md .Description md>
+source_url: github.com/YOUR_ORG/<md .Name md>
+access: private
+type: application
+
+app:
+  envs: {}
+    # Use jq expressions to build environment variables from input params or connections
+    # LOG_LEVEL: .params.log_level
+    # MONGO_DSN: .connections.mongo_authentication.data.authentication.username + ":" + .connections.mongo_authentication.data.authentication.password + "@" + .connections.mongo_authentication.data.authentication.hostname + ":" + (.connections.mongo_authentication.data.authentication.port|tostring)
+  policies: []
+    # Use jq expressions to select policies from artifact security blocks
+    # - .connections.sqs.data.security.iam.subscribe
+
+params:
+  examples:
+    - __name: Development
+      application:
+        sku_name: S1
+        minimum_worker_count: 1
+        maximum_worker_count: 3
+    - __name: Production
+      application:
+        sku_name: P3v3
+        minimum_worker_count: 1
```

```suggestion
        minimum_worker_count: 2
```
azure suggests

> Use Basic or higher plans with two or more worker instances for high availability.

https://docs.microsoft.com/en-us/azure/architecture/framework/services/compute/azure-app-service/operational-excellence

---

The customer will be paying for those additional instances. If they want to spin up an app for dev/test, I don't think they'll want 2 instances running at double the cost. Under the guided config for app templates, I can set the recommended minimum worker count for production environments.

---

yeah i agree. this comment was just to change it in the production guided config

---

Oops didn't notice that. Thanks!